src/urh/dev/native/lib/cbladerf.pxd

```diff
@@ -95,11 +95,12 @@ cdef extern from "libbladeRF.h":
         int bladerf_get_bandwidth(bladerf *dev, bladerf_module module, unsigned int *bandwidth)
 
     IF BLADERF_API_VERSION >= 2:
+      int bladerf_set_bias_tee(bladerf *dev, bladerf_channel ch, bool enable)
+      int bladerf_get_bias_tee(bladerf *dev, bladerf_channel ch, bool *enable)
+
+    IF BLADERF_API_VERSION >= 1.91:
         int bladerf_set_frequency(bladerf *dev, bladerf_channel ch, uint64_t frequency)
         int bladerf_get_frequency(bladerf *dev, bladerf_channel ch, uint64_t *frequency)
-    ELIF BLADERF_API_VERSION >= 1.91:
-        int bladerf_set_frequency(bladerf *dev, bladerf_channel ch, unsigned int frequency)
-        int bladerf_get_frequency(bladerf *dev, bladerf_channel ch, unsigned int *frequency)
```

Comment:
Is it safe to remove this?

Reply:
I had to remove those two methods and set it up to: 
```
int bladerf_set_frequency(bladerf *dev, bladerf_channel ch, uint64_t frequency)
int bladerf_get_frequency(bladerf *dev, bladerf_channel ch, uint64_t *frequency)
```

When compiling with `BLADERF_API_VERSION >= 1.91` works. Thus, it's safe to replace it.